scylla/tests/integration/statements/prepared.rs

```diff
@@ -554,8 +555,10 @@ async fn test_skip_result_metadata() {
         }
 
         // Verify that server sends metadata when driver doesn't send SKIP_METADATA flag.
+        // If the result metadata extension is enabled, then driver ignores the setting on PreparedStatement
+        // and always sends the SKIP_METADATA flag, so NO_METADATA should be present iff extension is enabled.
```

Comment:
:memo: Oh, that's a big thing (ignoring the setting on `PreparedStatement`). I'm leaving this here not to forget about this later. Perhaps you document the change well enough in the following commits.